Importer/Interpreter/EntityDataInterpreter.php

```diff
@@ -200,17 +203,18 @@ protected function interpretAssociativeRow(array $columns)
      */
     protected function getInterpreter(FieldConfigurationInterface $fieldConfiguration)
     {
-        if ($fieldConfiguration instanceof LookupFieldConfiguration) {
-            return $this->lookupFieldInterpreter;
+        switch ($fieldConfiguration) {
```

Comment:
would be nice to refactor this switch away with some intelligent code handling. otherwise it could just grow and grow.
Out of scope for now tho'